config/checker_severity_map.json

```diff
@@ -327,6 +335,7 @@
   "performance-move-const-arg":                                 "MEDIUM",
   "performance-move-constructor-init":                          "MEDIUM",
   "performance-noexcept-move-constructor":                      "MEDIUM",
+  "performance-no-automatic-move":                              "LOW",
```

Comment:
Duplicated entry can be found under this line. Please keep only one entry for this checker.